Android/app/src/main/java/io/github/project_travel_mate/MainActivity.java

```diff
@@ -408,7 +468,7 @@ void showProfileOrTrip(String data) {
                 Log.v("trip id", tripID + " ");
                 Trip trip = new Trip();
                 trip.setId(tripID);
-                Intent intent = MyTripInfoActivity.getStartIntent(MainActivity.this,  trip, false);
+                Intent intent = MyTripInfoActivity.getStartIntent(MainActivity.this, trip, false);
```

Comment:
why this change?

Reply:
I haven't changed this, I just pressed ctrl+alt+L to format the code. What's wrong with this, Can you please tell me?

Reply:
We dont want any extra changes in PR which are unrelated to issue.

Reply:
??